diagnostic.md

```diff
@@ -7,40 +7,46 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:rrdaniels85/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+
+ls
+(Confirm that it got pulled in successfully)
+Then cd into the new repository
```

Comment:
Hey this is close, you would do these steps but we were looking for creating a new branch to work on so you don't work on the master branch.